node/dataSources/document.ts

```diff
@@ -0,0 +1,44 @@
+import { RESTDataSource } from 'apollo-datasource-rest'
+import { withMDPagination } from '../resolvers/headers'
+import { parseFieldsToJson } from '../utils'
+
+export class DocumentDataSource extends RESTDataSource<ServiceContext> {
+  constructor() {
+    super()
+  }
+
+  public getDocument = (acronym, id, fields) => this.get(
+    `${acronym}/documents/${id}?_fields=${fields}`
```

Comment:
can you pass these query parameters in the second argument of the `get`?

Reply:
Yes, I resolved it.